modules/ztp-create-factory-hub-cluster.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="preparing-the-factory-install-environment_{context}"]
+= Preparing the factory install environment
+
+.Prerequisites
+Before you can provision edge clusters at scale, you must meet the following prerequisites:
+
+=== Base
+
+* Deploy the {product-title} cluster with three control plane nodes following the guidance in the section "Deploying installer-provisioned clusters on bare metal". Alternatively you can use the technology preview Assisted Installer from link:https://cloud.redhat.com/[cloud.redhat.com] to create the cluster.
+** All Cluster Operators are available.
+** Cluster is reachable using a `KUBECONFIG` file.
+** The API, API-INT and ingress should be deployed on the DHCP external Network (factory network).
+
+=== Storage
+
+* Storage can be provided by installing the Local Storage Operator and by using local volumes. However it is recommended to use OpenShift Data Foundation. Deploy the OpenShift Data Foundation following the guidance in link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/deploying_openshift_data_foundation_using_bare_metal_infrastructure/index[Deploying OpenShift Data Foundation using bare metal infrastructure].
```

Comment:
We should make it clear that if the cluster is 3+ nodes, then they use ODF.  If it's a single node OpenShift cluster, they should use the Local Storage Operator.

Reply:
excellent point thanks